OpenRA.Mods.Cnc/UtilityCommands/ExtractMegContentsCommand.cs

```diff
@@ -0,0 +1,56 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.IO;
+using OpenRA.Mods.Cnc.FileSystem;
+
+namespace OpenRA.Mods.Cnc.UtilityCommands
+{
+	class ExtractMegContentsCommand : IUtilityCommand
+	{
+		string IUtilityCommand.Name { get { return "--extract-meg"; } }
+
+		bool IUtilityCommand.ValidateArguments(string[] args)
+		{
+			return args.Length == 3;
+		}
+
+		[Desc(@"D:\SteamLibrary\steamapps\common\CnCRemastered\Data\music.meg", @"D:\exported", "Extracts a meg file into the target directory")]
```

Comment:
Likewise here - we should avoid shipping this for political reasons. We can use the ingame asset browser as a testcase instead.

Reply:
I used these commands to verify my implementation actually worked and did something useful. I have removed that commit from this branch